sw/airborne/modules/osd_max7456_new/max7456_regs.h

```diff
@@ -0,0 +1,84 @@
+#ifndef MAX7456_REGS_H
+#define MAX7456_REG_H
```

Comment:
the two names should match.
also, what is the difference with the "old" version from the `modules/display` folder ?